[0m15:57:24.805187 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000253A511E840>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000253A7FEBE60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000253A65AFB60>]}


============================== 15:57:24.844169 | bb45b333-c6b2-4557-80c5-fa58c410cc77 ==============================
[0m15:57:24.844169 [info ] [MainThread]: Running with dbt=1.10.2
[0m15:57:24.845168 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\gusta\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt init', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:01:10.024355 [debug] [MainThread]: Starter project path: C:\Users\gusta\Documents\Dev\jornadadedados\workshop-airflow-dbt\1_local_setup\.venv\Lib\site-packages\dbt\include\starter_project
[0m16:01:10.051287 [info ] [MainThread]: 
Your new dbt project "workshop_datawarehouse" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:01:10.053825 [info ] [MainThread]: Setting up your profile.
[0m16:05:25.645187 [info ] [MainThread]: Profile workshop_datawarehouse written to C:\Users\gusta\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m16:05:25.645187 [debug] [MainThread]: Command `dbt init` succeeded at 16:05:25.645187 after 480.96 seconds
[0m16:05:25.645187 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000253A6EF1550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000253A6EF0EC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000253A6EF3680>]}
[0m16:05:25.645187 [debug] [MainThread]: Flushing usage events
[0m16:05:26.168361 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m11:26:59.293616 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FE81879820>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FE80ADC920>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FE83FF6BA0>]}


============================== 11:26:59.297770 | 4ade31f6-5b2a-45e4-8ae1-10dfe3b264cf ==============================
[0m11:26:59.297770 [info ] [MainThread]: Running with dbt=1.10.2
[0m11:26:59.298292 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\gusta\\.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'invocation_command': 'dbt deps', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m11:26:59.299140 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\gusta\Documents\Dev\jornadadedados\workshop-airflow-dbt\2_data_warehouse\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:26:59.299733 [debug] [MainThread]: Command `dbt deps` failed at 11:26:59.299733 after 0.11 seconds
[0m11:26:59.300254 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FE83F1C7A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FE8446C4D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FE8446CD40>]}
[0m11:26:59.300774 [debug] [MainThread]: Flushing usage events
[0m11:26:59.820367 [debug] [MainThread]: An error was encountered while trying to flush usage events
